documentation/modules/configuring/con-exposing-kafka-connect-api.adoc

```diff
@@ -0,0 +1,66 @@
+// This assembly is included in the following assemblies:
+//
+//assembly-config-kafka-connect.adoc
+
+[id='con-exposing-kafka-connect-api-{context}']
+
+= Exposing the Kafka Connect API
+
+[role="_abstract"]
+Use the Kafka Connect REST API as an alternative to using `KafkaConnector` resources to manage connectors.
+The Kafka Connect REST API is available as a service running on `_<connect_cluster_name>_-connect-api:8083`, where _<connect_cluster_name>_ is the name of your Kafka Connect cluster.
+The service is created when you create a Kafka Connect instance.
+
+You can add connector configuration as a JSON object.
+
+.Example curl request to add connector configuration
+[source,curl,subs=attributes+]
+----
+curl -X POST \
+  http://my-connect-cluster-connect-api:8083/connectors \
+  -H 'Content-Type: application/json' \
+  -d '{ "name": "my-source-connector",
+    "config": [
+    {
+      "class":"org.apache.kafka.connect.file.FileStreamSourceConnector",
+      "file": "/opt/kafka/LICENSE",
+      "topic":"my-topic",
+      "tasksMax": "4",
+      # ...
+    }
+  ]
+}'
+----
+
+The API is only accessible within the cluster.
+If you want to make the Kafka Connect API accessible to applications running outside of the Kubernetes cluster, you can expose it manually by using one of the following features:
+
+* Services of types KafkaConnect or KafkaConnector
+
+* Ingress resources
+
+* OpenShift Routes
+
+NOTE: The connection is insecure, so allow external access advisedly.
+
+If you decide to create services, use the service labels in the `selector` of the `_<connect_cluster_name>_-connect-api` service to configure the pods to which the service will route the traffic:
+
+.Selector configuration for the service
+[source,yaml,subs=attributes+]
+----
+  # ...
+  selector:
+    strimzi.io/cluster: my-connect-cluster <1>
+    strimzi.io/kind: KafkaConnect
+    strimzi.io/name: my-connect-cluster-connect
+  #...
+----
+<1> Name of the Kafka Connect custom resource in your Kubernetes cluster.
+
+You must also create a `NetworkPolicy` that allows HTTP requests from external clients.
```

Comment:
Maybe we show the config for network policy. Wasn't sure what the example should look like.

Reply:
```yaml
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: my-custom-connect-network-policy
spec:
  ingress:
  - from:
    - podSelector:
        matchLabels:
          app: my-connector-manager
    ports:
    - port: 8083
      protocol: TCP
  podSelector:
    matchLabels:
      strimzi.io/cluster: my-connect-cluster
      strimzi.io/kind: KafkaConnect
      strimzi.io/name: my-connect-cluster-connect
  policyTypes:
  - Ingress
```

This part:
```yaml
    - podSelector:
        matchLabels:
          app: my-connector-manager
```
is basically what the users should customize to match their application.

Reply:
Thanks. Added with callout